src/Core/OrganizationFeatures/OrganizationSubscriptions/UpgradeOrganizationPlanCommand.cs

```diff
@@ -216,9 +216,19 @@ await _organizationUserRepository.GetManyByOrganizationAsync(organization.Id,
 
         if (string.IsNullOrWhiteSpace(organization.GatewaySubscriptionId))
         {
-            paymentIntentClientSecret = await _paymentService.UpgradeFreeOrganizationAsync(organization,
-                newPlan, upgrade);
-            success = string.IsNullOrWhiteSpace(paymentIntentClientSecret);
+            try
```

Comment:
❓ Is this bug only happening when upgrading a free organization?

Reply:
This only happening for upgrading a free organization